ethcore/types/Cargo.toml

```diff
@@ -18,11 +18,9 @@ parity-snappy = "0.1"
 patricia-trie-ethereum = { path = "../../util/patricia-trie-ethereum" }
 rlp = "0.4.0"
 rlp_derive = { path = "../../util/rlp-derive" }
+rustc-hex = "1.0"
```

Comment:
sorry @dvdplm but we still had `rustc-hex` as dev-dependency :)

Reply:
Can it be `2.0` at least? Maybe some day in a 100 years we'll have the whole code base upgraded... :)

Q: what if you use `into_inner()` on the `Header` to get the `Vec<u8>`? Can you go that route to print as hex you think? (Hex representation, I'm slowly realizing, is the third hardest problem in computing...)

Reply:
haha,

Well, we could I suppose but what's your objective here that the `display impl` is `undocumented hex formatted` or the `hex` dependency for `ethcore-types`?

After seing this, it might be better with a method `pub fn to_hex()` instead of `display`

Reply:
> the `hex` dependency for `ethcore-types`

Mainly this. `common-types` is waaay too fat for my liking as it is.

Reply:
I had in mind something like this (untested):
```
impl std::fmt::LowerHex for Header {
    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
            for byte in self.clone().into_inner() {
                write!(f, "{:02x}", byte)?;
            }
        Ok(())
    }
}
```